ts/components/ProfileTabIcon.tsx

```diff
@@ -16,8 +16,9 @@ type Props = OwnProps & ReturnType<typeof mapStateToProps>;
  */
 class ProfileTabIcon extends React.PureComponent<Props> {
   public render() {
-    const { size, color, isExperimentalFeaturesEnabled } = this.props;
```

Comment:
if you decide for the first proposed approach (see comment below), it could be maintained having `isExperimentalFeaturesEnabled` to `false` until the switch into the profile section is restored

Reply:
The first approach (maintain experimental features logic/store) could be a better solution.
Since isExperimentalFeaturesEnabled comes from store I force it to false. See line 21

Reply:
maybe we could not force it to false so that, once the switch is restored, no other changes to the code are needed.

Reply:
This is to prevent to show the experimental tab icon and force the default one. 
According with MatteoDS we don't need to show experimental tab icon when there are not experimental features available. It could be confusing for users

Reply:
Do you mean for those having `isExperimentalFeatureEnjabled` to true into their store at the time the app version is updated with these changes? It makes sense.

Reply:
@valentinacala exactly! We don't touch this user preference, we just ignore it! So when we will introduce a new experimental features, these users will have it back again

Reply:
ok, sounds better than a forced migration of `isExperimentalFeatureEnjabled` to false